modules/ln_glgui/chat.scm

```diff
@@ -204,54 +204,59 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ))
 
 (define (glgui:chat-input g wgt type mx my)
-  (let* ((x (fix (glgui-widget-get-dyn g wgt 'x)))
+  (let* ((mx-fixed (fix mx)) ;; Q: Can `mx` or `my` ever not be fixnums?
```

Comment:
I don't know, but we use `fx` math later, which segfaults if it is not without pointing to where the error is, which is extremely annoying to debug.